requests-for-comments/2025-01-30-podcast-feedback.md

```diff
@@ -0,0 +1,258 @@
+
+### How do we balance the quirky, fun tone with serious content?
+
+**Recommendation:** We have an over-the-top jingle and potentially a puppet in the branding for the show. But the conversations are "fundamentally a business talk". We can potentially balance the two, but it can admittedly come across as awkward and make the content seem dull when the fast-paced intro segues into a conversation.
+
+### How long should we schedule for recording?
+
+**Recommendation:** Our existing 60 minute block works well for now. If we have external guests, we should try to get more content by extending to 90 minutes, where the first ~20 minutes is a pre-interview.
+
+### Should Bijan host the show?
+
+**Recommendation:** We likely have a lot of people internally at PostHog that would be better at hosting. If Bijan continues to host, he'll have to rapidly improve at asking questions and directing the conversations.
+
+### Should we continue with the podcast as a goal?
+
+**Recommendation:** This is a meta question, but it's worth asking. We have excellent written and social content. But it seems like we've struggled with audio and video content. Part of the reason for the podcast was to make an intentional effort to improve in this area. But it's conceivable that we're not currently well-suited to creating this kind of content. And if that's the case, it's worth considering pivoting away from the goal before sinking additional resources into it.
+
+## Lessons learned and actionable takeaways
+
+### Intros
+  - Start each episode with 1-2 highlights. What is the most counterintuitive thing anyone said?
+  - Intros are critical. The host should rehearse the intro until it sounds smooth. If you make a mistake, start over instead of continuing past it.
+  - Introduce the show, the topic, and the guests. We tried long intros, short intros, and no intro. There definitely _needs_ to be an intro, but it should be as concise as possible.
+
+### Cadence
+  - Release one episode every other week, alternating with the newsletter release.
+
+### Hosting and interviewing
+  - Ask short, direct questions. Long questions lead to long, verbose answers.
+  - Ask one question at a time. If you ask multiple questions, guests will have to figure out which to respond to.
+  - If there are multiple guests, ask questions to a specific guest. Otherwise there will be an awkward pause about who should respond.
+  - It needs to be clear and obvious who is hosting.
+  - Ask follow up questions to dive deeper, rather than just moving on to the next topic.
+
+### Content
+  - Avoid "this is what we did at PostHog" content. Instead, content should follow this format: "We did this... and this is how you should think about the problem more broadly."
+
+### Tone
+  - Avoid coming across as salesy. If you notice the word "PostHog" is coming up 100x in an episode, you did something wrong.
```

Comment:
I wonder if we should just make an explicit goal that podcasts should always have an external guest / expert? I suspect we'll always fall into this trap if it's just us.